### PR TITLE
Refactor: change node styling

### PR DESCRIPTION
**Pull request by sonyakim-dev**
## Summary
image

### What was changed?
- Changed node style

### Why were these changes made?
-

### CODE REVIEW

File: src/views/CourseMap/components/CourseGraph.tsx [hunk 1]
```diff
@@ -6,17 +6,29 @@ interface CourseGraphProps {
...
+}: CourseGraphProps) {
   const [state, setState] = useState({
     graph: {
       nodes: [
         { id: 1, label: 'CS31' },
         { id: 2, label: 'CS32' },
         { id: 3, label: 'CS33' },
         { id: 4, label: 'CS35L' },
-        { id: 5, label: 'PHYS1A' },
+        { id: 5, label: 'PHYS1longtexttesttest' },
```
Comment (jason2020) [inline src/views/CourseMap/components/CourseGraph.tsx:31]: I think it's a good idea to test this. The current text wrapping is not the prettiest, but it seems reasonable. I would test with longer strings with spaces from Figma like "PREPARATION FOR THE MAJOR" or "AMERICAN HISTORY AND INSTITUTIONS". Ideally, it matches the Figma text wrapping, but honestly, it's a very minor detail.

File: tempCodeRunnerFile.python [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python
+import random
+import argparse
+
+def main():
```
Comment (jason2020) [inline tempCodeRunnerFile.python:5]: hello 35L lol, wow this is nostalgic. We all looove shuf, it's a real fan favourite!

**Review by jason2020 — COMMENTED** [review]
Nice clean up + styling is nice!

File: tempCodeRunnerFile.python (hunk 2 above)
Comment (sonyakim-dev) [reply tempCodeRunnerFile.python:5]: oh why is this here lol

**Pull request closed without merging**